.github/labeler.yml

```
@@ -0,0 +1,8 @@
+area:frontend:
+  - 'frontend/**'
+area:backend:
+  - 'backend/**'
+type:docs:
+  - '**/*.md'
+type:infra:
+  - '.github/**'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)